ooiui/static/js/views/common/SVGView.js

```diff
@@ -227,56 +233,9 @@ var SVGView = Backbone.View.extend({
 });
 
 var SVGPlotView = SVGView.extend({
-  setModel: function(model) {
-    this.model = model;
-    this.reference_designator = this.model.get('reference_designator')
-    this.stream_name = this.model.get('stream_name')
-    this.variable = this.model.get('yvariable').join()
-    this.dpa_flag = "0";
-    /* NEED TO ADD THIS BACK
```

Comment:
@birdage  not sure who put that in there but it's working fine without it, it's been committed out for ages!